checker/src/main/java/org/checkerframework/checker/nullness/NullnessChecker.java

```diff
@@ -3,7 +3,8 @@
 /** A concrete instantiation of {@link AbstractNullnessChecker} using freedom-before-commitment. */
 public class NullnessChecker extends AbstractNullnessChecker {
```

Comment:
This class doesn't have a purpose any more. We could remove it and rename `AbstractNullnessChecker` to `NullnessChecker`.